lib/response.js

```diff
@@ -958,9 +959,24 @@ res.render = function render(view, options, callback) {
   };
 
   // render
-  app.render(view, opts, done);
+  app.render(view, varStack, done);
 };
 
+/**
+ * Add `options` to the top of the reponse's template variables stack.
+ * Creates the stack the first time called.
+ *
+ * @public
+ */
+res.addTmplVars = function addTmplVars(options) {
```

Comment:
As a place holder, it's good, but yeah, that doesn't track for the long haul. 

+1   We'll probably want to think of a better method name than addTmplVars